### PR TITLE
Basic setup

### PR DESCRIPTION
**Pull request by JenilBhavsar**

### CODE REVIEW

File: server/data.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const products = [
```
Comment (hs2310) [inline server/data.js:1]: Add doc comments

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    if (user === 'admin') {
+        req.user = { name: 'admin', id: 1 }
+        next()
+    } else {
+        res.status(401).send('Unauthorized')
+    }
+}
+
+app.use(auth)
+
```
Comment (hs2310) [inline server/index.js:22]: Add doc comments

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    const id = Number(req.params.productID)
+    const product = products.find(product => product.id === id)
+    
+    if (!product) {
+        return res.status(404).send('Product not found')
+    }
+
+    res.json(product)
+})
+
```
Comment (hs2310) [inline server/index.js:40]: Add doc comments

File: server/data.js (hunk 1 above)
Comment (JenilBhavsar) [reply server/data.js:1]: Done

**Pull request merged**